docs/configuration/application.rst

```diff
@@ -0,0 +1,62 @@
+---------------------------
+Configuring the Application
+---------------------------
+
+The application is fed ``genesis.json`` the first time it starts up
+via the ``InitChain`` ABCI message. There are three fields that
+the application cares about:
+
+- ``chain_id`` must be consistent on all nodes and distinct from all
+  other blockchains. This is used in the tx signatures to provide replay
+  protection from one chain and another
+- ``validators`` are the initial set and should be stored if the app
+  wishes to dynamically adjust the validator set
+- ``app_state`` contains a map of data, to set up the initial blockchain
+  state, such as initial balances and any accounts with special permissions.
+
+App State
+=========
+
+If the backend ABCI app is weave-based, such as ``mycoind`` or ``bns``,
+the app_state contains one key for each extension that it wishes
+to initialize. Each element is an array of an extension-specific
+format, which is fed into ``Initialized.FromGenesis`` from the
+given extension.
+
+Sample to set the balances of a few accounts:
+
+.. code-block:: json
+
+  "app_state": {
+    "cash": [
+      {
+        "address": "849f0f5d8796f30fa95e8057f0ca596049112977",
+        "coins": [
+          {
+            "whole": 88888888,
+            "ticker": "BNS"
+          }
```

Comment:
Maybe it is nicer to use the string notation for the coin in the documentation.

```json
"coins": [
    "8888888 BNS",
    "321.765 IOV"
]
```

Reply:
yes, good point